portfolio/src/main/webapp/style.css

```diff
@@ -194,3 +194,11 @@ textarea, input, select, label, button {
   font-family: inherit;
   font-size: 1.3em;
 }
+
+.hidden {
+  display: none;
+}
+
+.hidden > .lds-roller {
```

Comment:
The lds-roller styles fit in a separate css file that you can add another link tab in index.html. You may name it 'lds-roller-style.css'.

Reply:
That sounds good, I will add it